File: app/src/main/java/tk/shyamkumaryadav/backroid/MainActivity.java
package tk.shyamkumaryadav.backroid;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    Button get_started;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        get_started = (Button)findViewById(R.id.button);
        get_started.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View v) {
                Toast.makeText(getApplicationContext(), "Welcome", Toast.LENGTH_LONG).show();
                Intent login_activity = new Intent(MainActivity.this, loginActivity.class);
                startActivity(login_activity);
            }
        });
    }
}

File: app/src/main/java/tk/shyamkumaryadav/mob/MainActivity.kt
package tk.shyamkumaryadav.mob

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.google.firebase.auth.FirebaseAuth

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val fireAuth = FirebaseAuth.getInstance()
        if (fireAuth.currentUser != null){
            updateUI("IS_LOGIN")
        }
    }

    fun btnGetLogin(view: View) {
        updateUI("LOGIN")
    }

    fun btnGetSignUp(view: View) {
        updateUI("SIGN_UP")
    }

    private fun updateUI(name:String){
        val intent = when(name){
            "LOGIN" -> Intent(this, LoginActivity::class.java)
            "SIGN_UP" -> Intent(this, SignUpActivity::class.java)
            "IS_LOGIN" -> Intent(this, MobActivity::class.java)
            else -> throw Exception("404 Error")
        }
        startActivity(intent)
        finish()
    }
}